### PR TITLE
feat(Pipeline): display user associated to the latest version

### PR DESCRIPTION
**Pull request by cheikhgwane**
Display user associated to the latest version of a pipeline.

## Changes

Please list / describe the changes in the codebase for the reviewer(s).

- Display last version associated user, if available
- Change 2

## How/what to test
Upload a new pipeline version :  on a PipelineCard a section with the associated user should be displayed.

## Screenshots / screencast
Screenshot 2024-10-09 at 14 46 59

### CODE REVIEW

**Comment by qgerome** [comment]
Nice start. I'd increase a bit the margin between the <User /> and the text above. I would remove the "Last version by" that is confusing since it's in another edge of the card. When you work with a card layout, you have to keep the pieces of info together. Here, it just adds a string for nothing. What I propose it to put that <User /> inside a Tooltip that would add the "Last version uploaded on ...." WDYT?

**Comment by cheikhgwane** [comment]
> Nice start. I'd increase a bit the margin between the and the text above. I would remove the "Last version by" that is confusing since it's in another edge of the card. When you work with a card layout, you have to keep the pieces of info together. Here, it just adds a string for nothing. What I propose it to put that inside a Tooltip that would add the "Last version uploaded on ...." WDYT?

Yes, you're right. It will be better.

**Comment by qgerome** [comment]
I'd put the User avatar and a tooltip on it that says : "Last version uploaded on {} by {}"

File: public/locales/fr/messages.json [hunk 1]
```diff
@@ -215,6 +215,7 @@
   "Last run": "Dernière course",
   "Last status": "Dernier statut",
   "Last updated": "Dernière mise à jour",
+  "Last version uploaded by {{name}}": "Dernière version téléchargée par {{nom}}",
```
Comment (qgerome) [inline public/locales/fr/messages.json:218]: This is not going to work. The interpolation variable should not change.

**Review by qgerome — CHANGES_REQUESTED** [review]

**Review by qgerome — APPROVED** [review]

**Pull request merged**